todo/tests/integration/test_tasks_delete.py

```
@@ -1,42 +1,43 @@
-from todo.constants.messages import ApiErrors
-from todo.tests.fixtures.task import task_dtos
+from http import HTTPStatus
+from rest_framework.test import APIClient
+from todo.tests.fixtures.task import tasks_db_data
+from todo.tests.integration.base_mongo_test import BaseMongoTestCase
 
 
-class TaskDeleteAPIIntegrationTest(APITestCase):
-    def setUp(self):
-        self.task_id = task_dtos[0].id
+class TaskDeleteAPIIntegrationTest(BaseMongoTestCase):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)